lib/iruby/kernel.rb

```diff
@@ -33,9 +33,7 @@ def create_backend
 
     def run
       send_status :starting
-      while @running
-        dispatch
-      end
+      dispatch while @running
```

Comment:
I don't want to include this change due to the following reasons.

1. I think the original form is more readable than the one-line form.
2. This change is not essential for the purpose of this pull-request.

Would you please revert this change?

Reply:
Ok. I will revert

Reply:
done. Thank you for your code revew.

Reply:
Please push your new commit.

Reply:
oh!

Reply:
Thanks!